### PR TITLE
Reworking the placement of metatype groups

### PR DESCRIPTION
**Pull request by andrey-churkin**
### Changes

Create a separate module to store metatype groups in one location.

### Reason for changes

Ref: 113580

### Related tickets

Ref: 113580

### Tests

N/A

### CODE REVIEW

File: nncf/openvino/graph/metatypes/groups.py [hunk 1]
```diff
@@ -46,34 +47,38 @@
 ]
 
 
-QUANTIZABLE_OPERATIONS = [
+INPUTS_QUANTIZABLE_OPERATIONS = [
```
Comment (andrey-churkin) [inline nncf/openvino/graph/metatypes/groups.py:50]: @KodiaqQ @alexsu52 I noticed that `ov_metatypes.OVDeformableConvolutionMetatype` is missing here, but it is present in `LINEAR_OPERATIONS` list and consequently used in fused patterns. So, my question is whether this is expected behavior and we should not quantize the `DeformableConvolution` operation, or if it is a bug that should be fixed?"

File: nncf/onnx/graph/metatypes/groups.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+
+MATMUL_METATYPES = [onnx_metatypes.ONNXGemmMetatype, onnx_metatypes.ONNXMatMulMetatype]
+
+
+INPUTS_QUANTIZABLE_OPERATIONS = [
+    onnx_metatypes.ONNXConvolutionMetatype,
+    onnx_metatypes.ONNXDepthwiseConvolutionMetatype,
+    onnx_metatypes.ONNXConvolutionTransposeMetatype,
+    *MATMUL_METATYPES,
```
Comment (KodiaqQ) [inline nncf/onnx/graph/metatypes/groups.py:54]: IMHO, it's better to place this at the end of the list.

File: nncf/openvino/graph/metatypes/groups.py (hunk 1 above)
Comment (KodiaqQ) [reply nncf/openvino/graph/metatypes/groups.py:49]: Never met `DeformableConvolution` in the OpenVINO models. Need to an example.

**Review by KodiaqQ — APPROVED** [review]

File: nncf/openvino/graph/metatypes/groups.py (hunk 1 above)
Comment (l-bat) [reply nncf/openvino/graph/metatypes/groups.py:49]: It isn't a bug. `DeformableConvolution` should be fused but runtime doesn't support INT8 execution (ticket: 106633)

**Review by l-bat — APPROVED** [review]

**Review by kshpv — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**